src/Data/Time/Format/Human.hs

```diff
@@ -143,3 +143,60 @@ humanReadableTimeI18N' (HumanTimeLocale {..}) cur t = helper $ diffUTCTime cur t
             | weeks d   < 5  = weeksAgo future $ i2s (weeks d)
             | years d   < 1  = onYear thisYear
             | otherwise      = onYear previousYears
+
+
+-- | Spanish human time locale.
+spanishHumanTimeLocale :: HumanTimeLocale
+spanishHumanTimeLocale = HumanTimeLocale
```

Comment:
Just realized that without adding this to the exports, it's not doing much :)

WDYT of a new module, maybe `Data/Time/Format/Human.Locales` to hold this, and establish the pattern of adding more languages to that module?